ansible/roles/debops.discourse/defaults/main.yml

```diff
@@ -0,0 +1,686 @@
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../includes/global.rst
+
+
+# Discourse version configuration [[[
+# --------------------------------
+
+# These variables define what Discourse release branch will be used to install
+# Discourse. The :envvar:`discourse_release` variable is used to enable/disable
+# specific features dependent on the Discourse version, but does not correspond to
+# the actual specific version number.
+#
+# Both of these variables should be kept in sync.
+
+# .. envvar:: discourse__release_map [[[
+#
+# This is the dictionary that keeps track of supported Discourse releases per OS
+# distribution/release. Older OS releases might not have the required
+# environment support, for example old Ruby version.
+discourse__release_map:
+  'buster':  'v2.3.2'
+
+                                                                   # ]]]
+# .. envvar:: discourse__distribution_release [[[
+#
+# Specify the OS release which is used to select the version of Discourse to
+# install and APT package names.
+discourse__distribution_release: '{{ ansible_local.core.distribution_release
+                                  if (ansible_local|d() and ansible_local.core|d() and
+                                      ansible_local.core.distribution_release|d())
+                                  else ansible_distribution_release }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse_release [[[
+#
+# Specify the Discourse release to install or manage. The role checks what release
+# is currently installed and can perform automatic upgrade if the installed
+# release is older than the specified. Downgrades are not supported.
+discourse_release: '{{ discourse__release_map[discourse__distribution_release] }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# Application features [[[
+# ------------------------
+
+# .. envvar:: discourse__database [[[
+#
+# What database to use for Discourse instance? Supported options:
+#
+# - ``postgresql``: use PostgreSQL database
+#
+# The role expects the selected database server to be configured. See the
+# documentation of the :ref:`debops.postgresql_server` role for information about
+# its features.
+discourse__database: '{{ ansible_local.discourse.database
+                      if (ansible_local|d() and ansible_local.discourse|d() and
+                          ansible_local.discourse.database|d())
+                      else ("postgresql"
+                            if (ansible_local|d() and ansible_local.postgresql is defined)
+                            else ("mariadb-is-deprecated"
+                                  if (ansible_local|d() and ansible_local.mariadb is defined)
+                                  else "no-database-detected")) }}'
+
+                                                                   # ]]]
```

Comment:
Does Discourse support the MariaDB database as well? Otherwise, this variable could probably be either removed, or its value simplified a bit to not check for MariaDB presence.